main.go

```diff
@@ -2804,6 +2810,61 @@ The following options are recognized:
 		return nil
 	})
 
+	r.Dispatch("uuid", &Help{
+		Summary: "Generate a version 4 uuid value and write it to the specified path and/or key. If no key is provide the key `uuid` is used",
```

Comment:
The summary is used for the help list for all commands. This would be a bit long for that output, so move this down to Description. This summary can simply be `Generate a new UUIDv4`, which would make it similar to other generator summaries.